pom.xml

```diff
@@ -149,7 +149,6 @@
     </dependencies>
 
     <build>
-        <finalName>nexus-replication-plugin</finalName>
```

Comment:
@gd-tmagrys, documentation/user_doc.md refers to target/nexus-replication-plugin-bundle.zip file that will be renamed  without this line. Please update user documentation.